From c1ec804f8f8b2a15a360e9b001ba50d4d06a4408 Mon Sep 17 00:00:00 2001
From: Tal Yacobi <tyacobi@habana.ai>
Date: Mon, 09 Jun 2025 09:33:37 +0300
Subject: [PATCH] Revert "[SW-221571] drivers: set device init log to warning"

This reverts commit 568a80667a7d09e22e1022d061ca391ff1132b62.

Set the log level after device_init() back to error.

Change-Id: Id3b315f21dd5f5fdc458bf9dfe89ebb0a0766d63
---

diff --git a/include/zephyr/drivers/misc/common/common_utils.h b/include/zephyr/drivers/misc/common/common_utils.h
index 8b7b63a..97fe05d 100644
--- a/include/zephyr/drivers/misc/common/common_utils.h
+++ b/include/zephyr/drivers/misc/common/common_utils.h
@@ -224,7 +224,7 @@
 				    (const struct device *dev = DEVICE_DT_GET(d);                  \
 					int err = device_init(dev);                                \
 					if (err) {                                                 \
-						LOG_WRN("Failed deferred init of device: %s (%d)", \
+						LOG_ERR("Failed deferred init of device: %s (%d)", \
 							#d, err);                                  \
 				     }),                                                           \
 				    (NULL));                                                       \
